website/pages/blog/announcing-pagerduty-plugin.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Reducing Alert Fatigue with the PagerDuty Plugin
+tag: announcement
+date: 2022/12/21
+description: >-
+  PagerDuty is a popular incident management tool used by many organizations. With the new PagerDuty plugin released today,
+we hope to help provide organzations insights on their incident management data, and help them reduce alert fatigue.
+
+author: shimon
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+PagerDuty is a popular incident management tool used by many organizations. With the new PagerDuty plugin released today,
+we hope to help provide organzations insights on their incident management data, and help them reduce alert fatigue.
+
+Among other possible use-cases, you can find out how long it takes to acknowledge and resolve incidents in your oragnizations,
+which users acknowledge and work on most incidents, and how many incidents each service generates. The example
+queries below limit the data to the last 3 months, but you can of course change the time-frame, or write queries
+that show month-by-month progess in the metrics.
+
+## Getting Started with CloudQuery and PagerDuty
+
+- Pick a destination to download your PagerDuty data into (this guide provides example queries for PostgreSQL). 
+  You can find our list of supported destinations [here](docs/plugins/destinations/overview).
+- Create a [PagerDuty authorization token](https://support.pagerduty.com/docs/api-access-keys#section-generating-a-general-access-rest-api-key).
+- Follow our [quickstart](/docs/quickstart) guide, and create a `pagerduty.yml` configuration file:
+  
+  ```yaml
+  kind: source
+  spec:
+    name: "pagerduty"
+    path: cloudquery/pagerduty
+    version: "VERSION_SOURCE_PAGERDUTY" 
```

Comment:
Should blog posts have an ever-increasing-auto-version or a static one? How do we make sure contents stay up to date wrt new versions?

Reply:
Me neither, but not make-or-break, I think :)